### ISSUE TITLE
Question: how can I test the full model accuracy?

### ISSUE THREAD

**Issue by Sunt-ing** [raymin0223/fast_robust_early_exit#13]
Hi, I notice that the current scripts are all for early exit models, but I want to try full models to verify the accuracy drop caused by early exit. How shall I do that?

BTW, the full models in your paper actually mean fine-tuned models, right? 

Also, I guess the accuracies of full fine-tuned models are higher than the original models, right?

Thanks!

**Comment by raymin0223** [comment]
Hi @Sunt-ing,

(1) Current scripts inlcude all settings, full model, early-exiting of static-exiting, CALM and FREE. https://github.com/raymin0223/fast_robust_early_exit/blob/98d60c3296bc54af82c32a3955125f9a7c55847a/scripts/run_sum_bigpatent.sh#L32-L66

from L.32 to L.46 is for testing full models. (note that you don't need to set `deploy_scenario` as True, cause this argument is only for early-exiting methods and it gets too slow when it's True)   
If you want to test other three early-exiting methods, just add arguments below each comments (e.g., add arguments from L.49 to L.53 for our FREE models).

(2) Full models mean fine-tuned models, yes. If you mean original model as _pretrained models_, yes it should be because they are finetuned on each downstream tasks!

**Comment by Sunt-ing** [comment]
Thanks for your answering!
